docs/_guides/codewind-eclipse-quick-guide.md

```diff
@@ -146,7 +146,7 @@ Any changes that you make to your code are automatically built and redeployed by
 You now can work with the example calculator microservice.
 
 * Use the **Exposed Application Port** number from the Application overview tab.
-* Make sure to remove the `< >` symbol in the URL.
+* Make sure to remove the **< >** symbol in the URL.
```

Comment:
If this was part of a URL, the "<>" could have also be left in code phrase format.